addOns/retire/src/main/java/org/zaproxy/addon/retire/model/Repo.java

```diff
@@ -307,4 +309,58 @@ private static boolean isGoodCandidate(String version) {
         }
         return isAllZeros != 0; // Not a good value if all zero
     }
+
+    static int getRiskForSeverity(String severity) {
+        if (severity == null || severity.isEmpty()) {
+            severity = "medium";
```

Comment:
Could return immediately the expected value (it could also set this when parsing).